pkgs/development/libraries/clutter-gtk/default.nix

```diff
@@ -19,6 +28,12 @@ stdenv.mkDerivation rec {
   propagatedBuildInputs = [ clutter gtk3 ];
   nativeBuildInputs = [ meson ninja pkg-config gobject-introspection ];
 
+  postPatch = ''
+    # ld: malformed 32-bit x.y.z version number: =1
```

Comment:
I have no clue what this comment is about. I assume it fixes some kind of error, right?

Reply:
Exactly.